# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/SelfHosting.cpp
# Commit: 0c3169618f2f
# Full Hash: 0c3169618f2f8415ee2cc91816e2cc3a8af1693e
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-09-09 15:22:41
# Regressor Bug: 1720422
# File Overlap Count: 1
# Description:
#   Bug 1720422 - Expose a templatized TypedArray<T> class in JSAPI r=jonco,nbp
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119853
# ==============================================================================

diff -r 3005e92c10a3 -r 0c3169618f2f js/src/vm/SelfHosting.cpp
--- a/js/src/vm/SelfHosting.cpp	Wed Sep 08 23:46:51 2021 +0000
+++ b/js/src/vm/SelfHosting.cpp	Wed Sep 08 23:46:56 2021 +0000
@@ -1354,7 +1354,7 @@
   MOZ_ASSERT(source->length() == target->length());
 
   switch (target->type()) {
-#define INIT_TYPED_ARRAY(T, N)                                         \
+#define INIT_TYPED_ARRAY(_, T, N)                                      \
   case Scalar::N: {                                                    \
     if (!ElementSpecific<T, UnsharedOps>::initFromIterablePackedArray( \
             cx, target, source)) {                                     \